pkg/display/display.go

```diff
@@ -56,3 +67,22 @@ func Display(ctx *cli.Context, displayable *d.Displayable) error {
 		return fmt.Errorf("unsupported display style: %s", displayStyle)
 	}
 }
+
+// getDisplay gets the default display settings.
+func getDisplay(fullName string) string {
+	// The variable displaystyle sets the input command and the corresponding output format.
+	var style = [][]string{{"dashboard global", "graph"},
+		{"dashboard global-metrics", "graph"},
+		{"metrics top", "table"},
+		{"metrics linear", "graph"},
+		{"metrics list", "table"},
+		{"service list", "table"},
+		{"t", "graph"},
+		{"trace", "graph"}}
```

Comment:
I suggest moving it out of the function and use `map[string]string` to maintain the display style. The above comment seems to be inconsistent.